### ISSUE TITLE
Update map view wireframe

### ISSUE THREAD

**Issue by thischrisoliver** [codefordc/housing-insights#289]
#### Updates to make after 5/23 discussion: ####
- [x] add charts drawer
- [ ] add export link/button
- [ ] categorize filters
- [ ] update mobile layout to include the side panels as different views (navigation for views could be dropdown at top or tab bar at bottom)

**added:**
- [ ] Create legend for categorical contextual layer
- [ ] Propose updated names for things that might not be understood by first time user (eg 'contextual layer', Filter = Filter Buildings ?, is "zones" from our current live site better than "boundaries" or not)?
- [ ] Check against https://github.com/codefordc/housing-insights/issues/252 to make sure we covered everything.

**Comment by thischrisoliver** [comment]
@NealHumphrey I added the charts drawer:
https://app.moqups.com/neal@nhumphrey.com/dDYbZTfu6T/edit/page/afbb51730

**Comment by NealHumphrey** [comment]
Thanks @thischrisoliver! I updated your description above to use checkboxes and checked off the charts drawer, plus added two additional bullet points we discussed Tuesday. 

@ostermanj @ptgott @MaryColley @wassona @Nunie123  for some additional context, on Tuesday the Filter team discussed the mockup linked above compared to our current view. We decided a few key things, most of which are reflected in the mockup but reiterating key points here so people know what we're building:
- We'll use two sidebars, left hand side for 'controls' (filtering/browsing) and right hand side for 'reports' (graphs, lists). 
- Each sidebar will have two modes selectable by a button / tab. Left hand side is a filter mode and a browse mode. Right hand side is 1) a list of filtered buildings and 2) a set of graphs. The graphs are divided into two parts, those related to building data and those related to browse data. 
- Color scheme ties related components together (e.g. an orange palette for anything building related, including graphs, and a blue palette for anything browse related). 
- Mobile view will be that every one of these 5 views (4 sidebars and the map) becomes a full screen thing that can be toggled between.
